package.json

```diff
@@ -0,0 +1,5 @@
+{
+  "dependencies": {
+    "node-html-parser": "^3.1.5"
+  }
+}
```

Comment:
Why do we need this here?

Reply:
Because I'm scrapping the trending section so I need that to parse the html

Reply:
I mean, why on the root? We keep root clean from these files.

Reply:
😬seems like a blunder from my side will fix it asap